tasks/affective_task/utils/timer.py

```diff
@@ -9,7 +9,7 @@
 REFRESH_RATE = 20
 
 
-def timer(seconds: int, callbacks: List[Callable], pre_text: str, screen):
+def timer(seconds: int, callbacks: List[Callable], pre_text: str, screen,display_timer = True):
```

Comment:
This feels misleading: if the user sets False to display_timer, the user expects that the screen does not show the timer at all, but here the timer is displayed after the time is halved.

You should consider having the argument "display_mode" instead, with 0 being show no timer, 1 is show timer, and 2 is show timer half the time.

Reply:
Done